test/OpenSTValue.js

```diff
@@ -300,7 +300,8 @@ contract('OpenSTValue', function(accounts) {
 		})
 
 		it('fails to process when msg.sender != staker', async () => {
-            await Utils.expectThrow(openSTValue.processStaking(stakingIntentHash, { from: registrar }));
+			// registrar can additionally as a fallback process staking in v0.9
```

Comment:
The `it` of this test should be updated to reflect what the comment implies—this tests that a non-staker/non-registrar cannot successfully call the function.

Reply:
changed to " fails to process when msg.sender is not staker or registrar"